services-js/commissions-search/src/FacetList.js

```diff
@@ -25,6 +25,7 @@ function renderAreaCheckbox(props, policyArea) {
     <Checkbox
       key={`checkbox-area-${policyArea.id}`}
       name={policyArea.id}
+      nameval={labelText}
```

Comment:
I'd recommend using `data-` for custom attributes. (E.g. `data-policy-name={labelText}`)